src/reducers/entities/gifs.ts

```diff
@@ -4,9 +4,10 @@
 import {combineReducers} from 'redux';
 import {GifTypes} from 'action_types';
 import {GenericAction} from 'types/actions';
-import {Dictionary} from 'types/utilities';
 
-const SEARCH_SELECTORS: Dictionary<Function> = {
+type ReducerMap = {[actionType: string]: (state: any, action: GenericAction) => any};
```

Comment:
Might be worth making this a utility type rather than just for this reducer.

Reply:
Yeah, I was thinking about that. I do kind of like this way of writing reducers instead of using a big switch statement, but I'm not wanting to make any big changes right now